terraform_utils/resource.go

```diff
@@ -86,7 +86,16 @@ func (r *Resource) ConvertTFstate() {
 	// delete empty array
 	for key := range r.InstanceState.Attributes {
 		if strings.HasSuffix(key, ".#") && r.InstanceState.Attributes[key] == "0" {
-			delete(attributes, key)
+			allowEmptyValue := false
+			for _, allowedEmptyValue := range r.AllowEmptyValues {
+				if allowedEmptyValue == key {
```

Comment:
`allowedEmptyValue` it's regex pattern.
Maybe better move `AllowEmptyValues` check to separate function and call to function from 2 places

Reply:
Well seen, done!